### ISSUE TITLE
Make accessibility.signalOptions.delays an experimental setting and count usage

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#213623]
Testing #213344

I am not convinced about the need for `accessibility.signalOptions.delays`. Can we mark it as experimental, and make sure to opt in to telemetry for counting how many users use this.

My suggestion - if we see more than 20 users using it after a month then we can consider to finalize it. If not, we can remove.

fyi @hediet

**Comment by meganrogge** [comment]
Roberto had suggested this should be configurable https://github.com/microsoft/vscode/issues/204257#issuecomment-2072394714

It is a good idea to validate that with telemetry 👍🏼

**Comment by meganrogge** [comment]
I am not able to see telemetry for specific properties of `signalOptions`. It doesn't look like there's a way to do this.

**Comment by connor4312** [comment]
@meganrogge I think you would have to manually event telemetry events for usage.

**Comment by meganrogge** [comment]
thanks @connor4312, I tried that by adding:

```
	if (e.affectsConfiguration('accessibility.signalOptions')) {
				this._telemetryService.publicLog2<AccessibilitySignalDelayConfigurationChangeClassification>('changedSignalOptions');
			}
```

However, it still doesn't indicate of the specific property, `delays`, changed. 🤔

**Comment by meganrogge** [comment]
@hediet another reason we should destructure the object #213618
